src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,526 @@
+        back_to_index: "Back to All"
+      auth:
+        sign_in: "Sign In"
+        sign_in_success: "You're now signed in"
+        sign_in_failure: "Sign in failed"
+        sign_up: "Sign up"
+        sign_up_success: "Thanks for signing up"
+        sign_up_failure: "Couldn't sign you up"
+        signed_out: "You have been signed out"
+        pwd_update: "Update Password"
+        pwd_update_success: "Your password has been reset"
+        pwd_reset: "Password Reset"
+        pwd_reset_request: "Reset your Password"
+        pwd_reset_req_success: "You should receive an email on how to reset your password shortly"
+      auth_token:
+        not_authenticated: "Not Authenticated."
+        invalid: "The provided authentication token was incorrect."
+        missing: "An authentication token is required. Please include a token in an 'auth_token' param or 'Authorization' header."
+      default:
+        page_name: Welcome
+      error:
+        title: "Something went wrong"
+        try_home: "Try heading back to home"
+        locked_out: "Locked-out"
+        auth_incorrect: "is wrong"
+        form_not_valid: "It looks like the form is not valid"
+        not_in_system: "is not in our system"
+      user:
+        email: E-Mail
+        profile: Profile
+        next: "Next, you may want to"
+        auth_guides: "Check out the 'Authentication Guides'"
+        modify_page: "Modify this page"
+        after_signin: "Change where you go after sign in"
+        preferred_language: "Preferred Language"
+    ```
+
+    > Add additional langauges as needed in the same folder.
+    > Be sure that all lang yml files contain the same keys. If there's no translation for that key, just leave the value blank.
+
+    ## Step 3 - Configure i18n within Lucky
+    ```
+    # config/i18n.cr
+    I18n.backend = I18n::Backends::YAML.new.tap do |backend|
+      backend.load_paths << Dir.current + "/config/locales"
+      backend.load
+    end
+    ```
+
+    ## Step 4 - Add 'lang' to users table
+
+    Generate a migration using:
+    ```
+    lucky db.migration AddLanguageToUser
+    ```
+
+    edit the file to look like (of course, the number will vary):
+    ```
+    # db/migrations/20191228100116_add_language_to_user.cr
+    class AddLanguageToUser::V20191228100116 < Avram::Migrator::Migration::V1
+      def migrate
+        alter table_for(User) do
+          add lang : String, default: "en"  # the appropriate default lang key
+        end
+      end
+
+      def rollback
+        alter table_for(User) do
+          remove :lang
+        end
+      end
+    end
+    ```
+
+    Of course migrate
+    ```
+    lucky db.migrate
+    ```
+
+    ## Step 5 - Add lang column to User model
+    ```
+    # src/models/user.cr
+    class User < BaseModel
+      # ...
+      table do
+        column lang : String
+        # ...
+      end
+      # ...
+    end
+    ```
+
+    ## Step 6 - Create a Translator module
+
+    ```
+    # src/translator.cr
```

Comment:
hmm.. Is this where this file should go? Maybe... I can't think of a better spot at the moment, but let's keep that in mind.

Reply:
OK - maybe:  `src/mixins/translators.cr` or something like that - would that be better?

Reply:
Maybe @paulcsmith might have an idea? I'm horrible with organization 😂 It was just my first thought felt like the root of the app was a bad place. I do kind of like your suggestion though.

Reply:
I agree - but - being new to this code - I had no idea on the standards - but I've used mixins in Rails - so its what came to mind - I could see components, etc.  But not sure if there are plans for an pluggable system and wouldn't want to mix the concepts

Reply:
OK - I updated it to use the folder src/mixins & (I clarified the yml files).  If Paul has another foldername in mind that's easy to change.

Reply:
I'm reverting this back to `src/translator.cr` with the recommendation from Paul as the location for central extensions.